docs/variants/tuxedo_ibs15/installation.md

```diff
@@ -0,0 +1,61 @@
+# Dasharo for Tuxedo IBS15 Gen6 - Installation manual
```

Comment:
Another very generic content which most of the time would be reused.

Reply:
Most of it is often reusable, yes. However, each device may have some specific caveats that affect it and only it. For example:

- Command for flashing will change after an update enables vboot
- TGL machines need an unreleased (for now) flashrom version

this is why i thought having a separate installation guide per device is a good idea.

Reply:
There would be additional effort fort unification. We should maximize use of common content and minimize customized one. Also we should only use customized content when it really is needed not just in case it will be needed in future.

I'm resolving that discussion, but guts tell me not to do that. I hope I will not have to get back to the same rethorics.